stable/admin/templates/role.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if or (not (hasKey .Values.nuodb "addRoleBinding")) .Values.nuodb.addRoleBinding }}
```

Comment:
this is a weird check, are there two ways to set it?

Reply:
I put this in because I want roles to be configured even if someone deletes the `addRoleBinding` field from the values file.